packages/@aws-cdk/aws-synthetics/README.md

```diff
@@ -102,30 +103,32 @@ Using the `Code` class static initializers:
 
 ```ts
 // To supply the code inline:
-const canary = new Canary(this, 'MyCanary', {
+new Canary(this, 'Inline Canary', {
   test: Test.custom({
-    code: synthetics.Code.fromInline('/* Synthetics handler code */'),
+    code: Code.fromInline('/* Synthetics handler code */'),
     handler: 'index.handler', // must be 'index.handler'
   }),
-  runtime: synthetics.Runtime.SYNTHETICS_NODEJS_PUPPETEER_3_1,
+  runtime: Runtime.SYNTHETICS_NODEJS_PUPPETEER_3_1,
 });
 
 // To supply the code from your local filesystem:
-const canary = new Canary(this, 'MyCanary', {
+new Canary(this, 'Asset Canary', {
   test: Test.custom({
-    code: synthetics.Code.fromAsset(path.join(__dirname, 'canary')),
+    code: Code.fromAsset(path.join(__dirname, 'canary')),
     handler: 'index.handler', // must end with '.handler'
   }),
-  runtime: synthetics.Runtime.SYNTHETICS_NODEJS_PUPPETEER_3_1,
+  runtime: Runtime.SYNTHETICS_NODEJS_PUPPETEER_3_1,
 });
 
 // To supply the code from a S3 bucket:
-const canary = new Canary(this, 'MyCanary', {
+import * as s3 from '@aws-cdk/aws-s3';
+const bucket = new s3.Bucket(this, 'Code Bucket');
+new Canary(this, 'Bucket Canary', {
   test: Test.custom({
-    code: synthetics.Code.fromBucket(bucket, 'canary.zip'),
+    code: Code.fromBucket(bucket, 'canary.zip'),
```

Comment:
keep the `synthetics` prefix, there is a `Code` class in `aws-lambda` , the prefix prevent the confusion. 
Now that v2 is out its a good opportunity to think about consolidating the different `Code` classes.